curiouslearning/FeedTheMonsterJS
# feat: Use the base button component and functionality (FM-296)

# Changes
- Update buttons in the gameplay UI to html

# How to test
- Open the start screen of FTM.
- Tap or click any part of the start screen to proceed to the next screen
- Select any level to play
- Make sure the buttons are working the same
- To test the html buttons for example in cancel button
```
// Standard instantiation with defaults
const yesButton = new YesButtonHtml();

// Custom instantiation with overrides
const customYesButton = new YesButtonHtml({ id: 'custom-yes-button', className: 'custom-yes-class' });
```

- To add custom functions
```
const pauseButton = new PauseButton();
pauseButton.onClick(() => {
  console.log('Game Paused');
});
```

Ref: [FM-296](https://curiouslearning.atlassian.net/browse/FM-296)


[FM-296]: https://curiouslearning.atlassian.net/browse/FM-296?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
